src/App.js

```diff
@@ -1,35 +1,40 @@
-import React, { Component } from 'react';
-import Header from './Header.js';
-import request from 'superagent';
-import PokeSearch from './PokeSearch.js';
-import PokeDisplay from './PokeDisplay.js';
-import './App.css'
-export default class App extends Component {
-  state = {
-    search: '',
-    isLoading: false,
-    pokeState: []
-  }
+import React, { Component } from 'react'
+import {
+    BrowserRouter as Router, 
+    Route, 
+    Switch,
+    
+} from 'react-router-dom';
 
-  handleClick = async () => {
-    this.setState({ isLoading: true })
-    const pokeData = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?perPage=1000&pokemon=${this.state.search}`);
-    this.setState({
-      pokeState: pokeData.body.results,
-      isLoading: false,
-    })
-    console.log(pokeData)
-  }
+import SearchPage from './SearchPage/SearchPage.js';
+import DetailPage from './DetailPage/DetailPage.js';
 
-  searchValue = (e) => this.setState({search: e.target.value})
-  render() {
-    return (
-      <div className='main-box'>
-        <Header/>
-        <PokeSearch search={this.searchValue} click={this.handleClick} />
-        <PokeDisplay  pokeState={this.state.pokeState}/>
-      </div>
-    )
-  }
-}
 
+export default class App extends Component {
+    state = {
+        search: '',
+        isLoading: false,
+        pokeState: []
+    }
+    render() {
+        return (
+            <div>
+                      <Router>
```

Comment:
Nit-picky comment here- but make sure to use the correct indentation and remove extra whitespace in your code!